clickable/clickable.yaml

```diff
@@ -0,0 +1,40 @@
+clickable_minimum_required: '8.1.0'
+framework: ubuntu-sdk-20.04
+builder: cmake
+ignore_review_errors: true
+
+build_args:
+- DNCINE_APP_NAME='Jazz Resurrection'
+
+dependencies_target:
+- libasound2
+- libdw-dev
+- libdwarf-dev
+- libfreetype6
+- libgl1-mesa-dev
+- libglew-dev
+- libglfw3-dev
+- libglib2.0-0
+- libopenal-dev
+- libopenmpt-dev
+- libpng16-16
+- libsdl2-dev
+- libvorbisfile3
+- zlib1g-dev
+
+install_lib:
+- libGLEW.so*
+- libglfw.so*
+- libdw*.so*
+- libOpenGL.so*
+
+install_root_data:
+- clickable/jazz2.apparmor
+- clickable/jazz2.desktop
+- clickable/jazz2.png
+- clickable/manifest.json
+
+postbuild:
```

Comment:
The postbuild step could be avoided, if the "bin" path wasn't hard-coded in https://github.com/xdub/jazz2-native/blob/master/cmake/ncine_installation.cmake#L196. Options for changing this:

- Use [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#module:GNUInstallDirs) and pass `-DCMAKE_INSTALL_BINDIR=${CLICK_PATH}` in the `build_args`
- Add an option to the `CMakeLists.txt` and configure it in the `build_args`
- Make the install cmake script aware of click builds and let it take `CLICK_PATH` from the environment